functions/src/application/stats.ts

```diff
@@ -0,0 +1,63 @@
+// Third party modules
+import * as firebase from 'firebase-admin';
```

Comment:
I think we can use `functions/src/client/firebase-admin.ts` to make it easier. There might be some missing `export`s that need to be added